bin/helm_toolbox.sh

```diff
@@ -17,40 +19,32 @@ function check() {
 	command -v "$1" >/dev/null 2>&1 || err "$1 not installed!"; 
 }
 
-# wait for helm to become operational
-wait_for_helm() {
-	info "Waiting for helm tiller..."
-	while true; do
-		status=$(echo $(kubectl get pods -l app=helm -l name=tiller --show-all=false -o=custom-columns=STATUS:.status.phase --no-headers=true -nkube-system))
-		info "Helm status: $status"
-		if [ "$status" = "Running" ]; then
-			break;
-		fi
-		sleep $TIMEOUT
-	done
-}
-
 function set_context() {
   if [ -n "$KUBE_CONTEXT" ]; then
 	info "Using ${KUBE_CONTEXT} kube context"
 	kubectl config use-context ${KUBE_CONTEXT}
   fi
 }
 
+function retry() {
+  attempt=1
+  until [[ $n -ge $2 ]]
+  do
+	info "Perform attempt - $attempt"
+    $1 && break
+    n=$[$attempt]
```

Comment:
Related to previous comment, I don't think we want `$n` here but instead something like: `attempt=$[$attempt+1]`

Reply:
sorry, mistyped.
fixed now.